pkg/local_object_storage/shard/gc.go

```diff
@@ -431,8 +431,17 @@ func (s *Shard) HandleExpiredLocks(lockers []oid.Address) {
 		return
 	}
 
+	expired, err := s.metaBase.FilterExpired(unlocked)
+	if err != nil {
+		s.log.Warn("expired object filtering",
+			zap.Error(err),
+		)
+
+		return
+	}
+
 	var pInhume meta.InhumePrm
-	pInhume.SetAddresses(lockers...)
+	pInhume.SetAddresses(append(lockers, expired...)...)
```

Comment:
lets reflect this in method docs? same for `HandleDeletedLocks`

Reply:
done